TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.1.7"
+          "version": “1.1.8-servicing-*”
```

Comment:
these quotes look a little funny. Please make sure they don't cause issues.

Reply:
Good call, fixed. Think it's something funky from the text editor I was using on my mac.